Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1092,7 +1092,9 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         // Making sure that we correctly let decoding memory get released, do not remove the autorelease pool.
         try autoreleasepool {
             /// We need only unique relationships so we'll collect them in a set.
-            var combinedRelationships = [UnifiedSymbolGraph.Selector: Set<SymbolGraph.Relationship>]()
+            var combinedRelationshipsBySelector = [UnifiedSymbolGraph.Selector: Set<SymbolGraph.Relationship>]()
+            /// Also track the unique relationships across all languages and platforms
+            var uniqueRelationships = Set<SymbolGraph.Relationship>()
```

Comment:
Part of me worries that this will create a situation where some relationships show up in language views where they otherwise shouldn't, but the existing code already doesn't account for this, so it's no worse than today. 🤷‍♀️

Reply:
Yes probably we need to work on the code inside of `createInheritedSymbolsAPICollections` to handle this in a language-aware manner. But this fix at least removes the duplicates we have now.